avogadro/qtplugins/noncovalent/noncovalent.cpp

```diff
@@ -21,10 +23,17 @@
 #include <QtWidgets/QVBoxLayout>
 #include <QtWidgets/QWidget>
 
+#include <cmath>
+
+#define M_TETRAHED 1.910633236
```

Comment:
I'm okay with these, but we should make clear where the numbers come from, either as a comment or a clear `M_TETRAHEDRAL = …` and have the compiler compute the constant.